examples/nginx-app/with-pv.yaml

```diff
@@ -43,7 +43,7 @@ metadata:
   name: nginx-deployment
   namespace: nginx-example
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
We "cheated" a bit in [here](https://github.com/heptio/ark/blob/master/docs/cloud-provider-specifics.md#snapshot-example-with-pvs) to label one node with `app=nginx` to ensure pods from the deployment all land on the same node. @abiogenesis-now I think it's cleaner this way, and then we can remove the node labeling bit from the docs. WDYT?